.devops/cstar-rtd-code-review-pipelines.yml

```diff
@@ -0,0 +1,77 @@
+# Build your Java project and run tests with Apache Maven.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/java
+
+pr:
+  autoCancel: false
+  branches:
+    include:
+      - main
```

Comment:
we mustn't include "main" and "uat" here, must we?

Reply:
i think it's pretty safe because it only triggers the code-review pipeline